### PR TITLE
Topic/release notes 3 7 0

### PR DESCRIPTION
**Pull request by crucialfelix**
Updated README files.
Also removes obsolete examples and fixes tabs vs spaces

Please read through and post your quibbles and comments.
Any changes for Linux or Windows please let me know and I can include them.

I'll merge this, but I'm interested in feedback.

### CODE REVIEW

**Comment by bagong** [comment]
Maybe this is the place to say it: I don't see a Windows 3.7 release happening. If nobody steps up that can actually do it (and not just moderate it, like me) it won't happen. I have given up maintaining my build system months ago and don't have the time to revive it fttb. Atm, from what I can see, there is no Windows version that builds, let alone works... Sorry.... Hopefully somebody can be found...

**Comment by crucialfelix** [comment]
The call for a volunteer has certainly been made several times. We won't hold up the 3.7 release for windows. I can note it in the build instructions. If somebody can fix the issues then we can issue a point release (doesn't break any features). 

@jamshark70 Don't you work on Window often ?

**Comment by telephon** [comment]
> I always build from command line.

I used to do that for most of the time, but then I couldn't navigate the project in xcode – it was very tedious, trying to do everything by "find" in the "finder". Is there a way to still build on command line even if you have configured the build for xcode?

File: README_OS_X.md [hunk 1]
```diff
@@ -123,83 +134,93 @@ While it's generally safe to re-use your build folder, changing branches, build
...
-manually editing build/CMakeCache.txt. 
+If you feel uncomfortable with the command line, you might want to try cmake
+frontends like  `ccmake` or `cmake-gui`. You can also configure your build by
+manually editing build/CMakeCache.txt.
 
 
 Common arguments to control the build configuration are:
 
-  * Control the location where SC get's installed. The following line moves it to the Applications folder 
+  * Control the location where SC get's installed. The following line moves it to the Applications folder
```
Comment (danstowell) [inline README_OS_X.md]: "get's" should be "gets"

Comment (danstowell) [inline README_OS_X.md]: (that's the only thing I could find to complain about in the pr...)

Comment (crucialfelix) [inline README_OS_X.md]: thanks

**Comment by miczac** [comment]
great work, thanks!
A question regarding OS X: is there an agreement about a minimum OS X version?
README_OS_X.md states "Any build from the 6.x series or greater should generally work."
That implies OS X 10.9 with XCode 6.1.1. 
A note on that would be great, IMHO.

**Comment by crucialfelix** [comment]
I wasn't sure about that myself. Maybe it does work all the way back to 10.7 and XCode 4.2.

travis is building with target: MAKE_OSX_DEPLOYMENT_TARGET=10.7
Travis CI uses OS X 10.9.5

I do know that I updated my XCode recently and it assumed that I was on El Capitan 10.11 (which I was valiantly resisting) and after that I couldn't compile anything. So I was forced to do the upgrade.

Can anybody here confirm building on 10.7 ?

**Comment by miczac** [comment]
hm, IFMP never had 10.7 and just upgraded from 10.8.5 to 10.9.5 (some simulator I need requires that).
And it gets a tad more complicated since supernova won't compile following the instructions as they are. Also not with with Xcode 6.2 (the newest one running on 10.9). 
For lack of resources I couldn't yet test if a newer Xcode would solve this (probably 6.3.1?). 
@timblechmann very briefly suggested [1] to use gcc, which alas yet is beyond my abilities to get the tool-chain cleanly adapted. 
[1] https://github.com/supercollider/supercollider/pull/1636#issuecomment-137896607
     interestingly enough I can't remember where I got from that Xcode 6.1 would work.
     In fact, it doesn't (just tried w/ 6.2 on 10.9.5)

**Comment by miczac** [comment]
> Travis CI uses OS X 10.9.5
well, I managed to test: 
Supernova compiles OK on Xcode >= 6.3 (clang 6.1) which itself requires OS X 10.10. 
I set a deployment target of 10.7 and compiled SC on a VM w/ 10.10 w/o problems.
Supernova compiled that way booted and sounded ok on native 10.9.  
So, what to do regarding 3.7b and its Readme?
Either it's increasing the minimum OS X version to 10.10 (oh well ...) or add gcc as another prerequisite (or one just for Supernova) and adapt the cmake configuration accordingly.
IMHO not that nice either way...

**Comment by crucialfelix** [comment]
*Xcode 6 requires a Mac running OS X version 10.9.4 or later or 10.10*
(you've proven that its compilable using xcode 6)


*Xcode 5 requires a Mac running OS X v10.8.4*
(we can say that it may work, no harm in trying)

**Comment by crucialfelix** [comment]
I'd like to merge this today. Can I get an Amen ?

The CHANGELOG.md section for 3.7.0 will be regenerated just before the release. But it would be similar to this: https://github.com/crucialfelix/supercollider/blob/topic/release-notes-3-7-0/CHANGELOG.md

The previous ChangeLog stays there for now, but all the contents have been moved into CHANGELOG.md  Or I can just remove it now.

I have not updated the News 3.7 schelp so its a bit thin. I am a bit worn out and I don't know what to put in there.

**Comment by danstowell** [comment]
You are a TROOPER Mr Cruxxial, thanks for all your work! I just gave you a PR with a couple of notes for the news, if you like.

**Comment by crucialfelix** [comment]
got it. merging now. will deal with a beta tag tomorrow. we can always add stuff to the news after the beta is out.

people have to kick the tires.

**Pull request merged**